cves/kernel/CVE-2015-4178.yml

```diff
@@ -131,10 +133,12 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: The original code did not contain any unit testing
```

Comment:
This makes it sound like tests were added later IMO. You should clarify whether or not this is the case.

Reply:
That makes sense, wording has been clarified.